front/src/modules/ui/table/editable-cell/components/EditableCell.tsx

```diff
@@ -84,6 +84,23 @@ export function EditableCell({
 
   const hasSoftFocus = useIsSoftFocusOnCurrentCell();
 
+  function EditButton() {
```

Comment:
Components should be in a standalone file. Maybe `EditableCellEditButton`

Reply:
Done!